www/docs/guides/L1message.md

```diff
@@ -36,7 +36,7 @@ You have to pay in the L1 an extra fee when invoking `sendMessageToL2` (of cours
 
 ```typescript
 import { SequencerProvider } from 'starknet';
```

Comment:
Replace `SequencerProvider` with `RpcProvider`. I believe this is the last reference in the non-versioned docs.

Reply:
Made.